homeassistant/components/freebox/router.py

```diff
@@ -175,10 +208,21 @@ async def update_home_devices(self) -> None:
 
         try:
             home_nodes: list[Any] = await self.home.get_home_nodes() or []
-        except HttpRequestError:
+        except InsufficientPermissionsError:
```

Comment:
Revert that, I don't remember exactly why now but catching `HttpRequestError` used to fix an error, check the line history

Reply:
Done. Only caching InsufficientPermissionsError exception for now

Reply:
```suggestion
        except HttpRequestError:
```